mongodbatlas/resource_mongodbatlas_advanced_cluster_test.go

```diff
@@ -327,6 +332,13 @@ func TestAccClusterAdvancedCluster_PausedToUnpaused(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "paused", "false"),
 				),
 			},
+			{
+				Config:      testAccMongoDBAtlasAdvancedClusterConfigSingleProviderPaused(orgID, projectName, rName, true, "M10"),
+				ExpectError: regexp.MustCompile("CANNOT_PAUSE_RECENTLY_RESUMED_CLUSTER"),
+			},
+			{
+				Config: testAccMongoDBAtlasAdvancedClusterConfigSingleProviderPaused(orgID, projectName, rName, false, "M10"),
+			},
```

Comment:
Did we test the same behaviour with the cluster resource? they shared common functions so we should make sure that the behaviour there is correct

Reply:
> I tried to fix the same for cluster but APIs and logic inside are different than advanced_cluster - a fix would probably make it worse instead of better.

I just say this comment. Could you elaborate a little more? Maybe we can file a ticket for the Atlas Cluster team if this is a problem with the API

Reply:
> I just say this comment. Could you elaborate a little more? Maybe we can file a ticket for the Atlas Cluster team if this is a problem with the API

I don't think there is a problem with the APIs v1.0, I am just saying that right now there is no issue with `cluster`, but if we make the change the new behaviour would introduce an unnecessary breaking change for the `cluster` resource. 

Do we want that? I'd say no for one reason: I don't think customers are either using `cluster` or `advanced_cluster`, not both concurrently. In addition to that, once they'll have to migrate to `advanced_cluster` it'll be a breaking change for them no matter what.

I am open if you have a different opinion but only thing I can do is to create a separate Jira issue to align `cluster` to this - from a trial I did (that you can see in the commit history) it wasn't an immediate change hence it requires careful and tested changes and I don't want to slow this PR down (related to a HELP issue).